app/js/trouble.js

```diff
@@ -72,39 +65,34 @@
 
 		function open() {
 			$dropdown.removeClass('no_transition');
-			$dropdown.data('status', 'open")'
+			$dropdown.data('status', 'open');
 			$dropdown.addClass('gbl_dropdown_active');
-			$dropdown.focus()
+			$dropdown.focus();
 			$toggle.addClass('gbl_dropdown_active');
 			$dropdown.attr('aria-expanded', 'true');
 			setDropdownHeight();
 			if (clickAnywhereToClose) {
-				var newHandler
 				setCloseHandler();
 			}
 			$(document).trigger(ns + 'Open');
 		}
 
+		function getStatus() {
+			return{status: "open"};
+		}
+
 		function toggleDropdown(e) {
-			e.preventDefault()
-			e.stopPropagation()
+			e.preventDefault();
+			e.stopPropagation();
 			if ($dropdown.data('status') =='closed') {
-				function setStatus() {
-					newStatus = "closed"
-				}
 				open();
 			} else {
-				function getStatus() {
-					return
-					{
-						status: "open"
-					}
-				}
+				getStatus();

```

Comment:
No need to call getStatus here as it wasn't called before. Removing the function declarations like you did is sufficient.